src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -157,15 +157,16 @@ export class CopyQueryWithResultsKeyboardAction extends Action {
 				let resultSummary = queryRunner.batchSets[0].resultSetSummaries[i];
 				let result = await queryRunner.getQueryRows(0, resultSummary.rowCount, resultSummary.batchId, resultSummary.id);
 				let tableHeaders = resultSummary.columnInfo.map((col, i) => (col.columnName));
-				let htmlTableHeaders = resultSummary.columnInfo.map((col, i) => (`<th style="border:solid black 1.0pt; whiteSpace:nowrap">${escape(col.columnName)}</th>`));
+				let htmlTableHeaders = resultSummary.columnInfo.map((col, i) => (`<td style="border:1.0px solid black;padding:3pt;font-size:9pt;font-weight: bold;">${escape(col.columnName)}</td>`));
```

Comment:
why are we using mixed px and pt?

Reply:
This style was taken from what latest Kusto Explorer code has. I can change it if required